pom.xml

```diff
@@ -405,6 +405,42 @@
                  <goal>verify</goal>
               </goals>
             </execution>
+            <execution>
+              <id>SignOffTests</id>
+              <configuration>
+                <reuseForks>false</reuseForks>
+                <argLine>@{jacocoArgLine} -enableassertions -Xmx3000m -Djava.security.egd=file:/dev/./urandom "-Djava.library.path=${hadoop.library.path}${path.separator}${java.library.path}" -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=./target/ -XX:NewRatio=4 -XX:SurvivorRatio=8 -XX:+UseCompressedOops -XX:+UseConcMarkSweepGC -XX:+DisableExplicitGC -XX:+UseCMSInitiatingOccupancyOnly -XX:+CMSClassUnloadingEnabled -XX:+CMSScavengeBeforeRemark -XX:CMSInitiatingOccupancyFraction=68</argLine>
+                <groups>org.apache.phoenix.end2end.SupportsDistributedClusterTest</groups>
```

Comment:
What category is SupportsDistributedClusterTest? How is it different from SuccessfulSupportsDistributedClusterTest? 
Or is the intent to use one or the other? In that case I would vote for "SupportsDistributedClusterTest"

Reply:
@jpisaac the SuccessfulSupportsDistributedClusterTest is the category for all the successful tests. the SupportsDistributedClusterTest  was a category i was using to test the IT's I hadnt tested and then categorizing them into Successful/Failure distributed cluster test